src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -51,6 +51,8 @@
 #include <mlpack/methods/ann/layer/parametric_relu.hpp>
 #include <mlpack/methods/ann/layer/radial_basis_function.hpp>
 #include <mlpack/methods/ann/layer/relu6.hpp>
+#include <mlpack/methods/ann/layer/repeat.hpp>
+#include <mlpack/methods/ann/layer/replicate.hpp>
```

Comment:
We should also add these two layers to `serialization.hpp` too.

Reply:
👍